examples/nextjs/starter/src/app/sitemap.ts

```diff
@@ -0,0 +1,17 @@
+import type { MetadataRoute } from 'next';
+import {
+	generateSitemaps as generateWPSitemaps,
+	generateSubSitemaps,
+} from '@snapwp/next';
+
+export async function generateSitemaps() {
+	return await generateWPSitemaps();
+}
```

Comment:
Will a re-export work here?
```suggestion
export generateSitemaps
```

Reply:
Not resolved